University of Wisconsin-Madison Libraries Catalog.js

```diff
@@ -0,0 +1,256 @@
+				"publisher": "Chicago : Association of College and Research Libraries, 2011.",
+				"shortTitle": "Zotero",
+				"url": "https://search.library.wisc.edu/catalog/9910104568102121",
+				"attachments": [],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Includes bibliographical references.</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://search.library.wisc.edu/catalog/9912334246702121",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Fiscal year 2017 budget amendments : communication from the President of the United States transmitting FY 2017 budget amendments for national security activities at the Department of Defense, the Department of State, and the U.S. Agency for International Development to fund overseas contingency operations",
+				"creators": [
+					{
+						"lastName": "United States. President (2009-2017 : Obama)",
+						"creatorType": "author",
+						"fieldMode": 1
+					}
+				],
+				"date": "2016",
+				"abstractNote": "38 pages ; 24 cm",
+				"callNumber": "Y 1.1/7: 114-178",
+				"libraryCatalog": "University of Wisconsin-Madison Libraries Catalog",
+				"publisher": "Washington : U.S. Government Publishing Office, 2016.",
+				"shortTitle": "Fiscal year 2017 budget amendments",
+				"url": "https://search.library.wisc.edu/catalog/9912334246702121",
+				"attachments": [],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>Shipping list no.: 2017-0006-M.;&quot;November 14, 2016.&quot;;&quot;Referred to the Committee on Appropriations.&quot;;Microfiche. [Washington, D.C.] : U.S. Government Printing Office, [2017] 1 microfiche : negative.</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://search.library.wisc.edu/catalog/999618880602121",
+		"items": [
+			{
+				"itemType": "audioRecording",
+				"title": "Settimo in mi bem. magg., op. 20 = Septet in E flat major, op. 20",
+				"creators": [
+					{
+						"lastName": "Beethoven",
+						"firstName": "Ludwig van",
+						"creatorType": "composer"
+					}
+				],
+				"date": "[1988]",
+				"abstractNote": "1 audio disc (43 min.) : digital ; 4 3/4 in",
+				"callNumber": "COMP DISC 385",
+				"label": "Pontelambro, Como, Italy : Nuova Era, [1988] ℗1988",
+				"libraryCatalog": "University of Wisconsin-Madison Libraries Catalog",
+				"url": "https://search.library.wisc.edu/catalog/999618880602121",
+				"attachments": [],
+				"tags": [],
+				"notes": [
+					{
+						"note": "<p>For clarinet, horn, bassoon, violin, viola, violoncello, and double bass.;Compact disc.;Duration: 42:10.;Program notes by Paolo Petazzi in Italian and English ([14] p. : ill.) inserted in container.;Forms part of: Curtiss Blake Collection.</p>"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://search.library.wisc.edu/catalog/9910065379202121",
+		"items": [
+			{
+				"itemType": "audioRecording",
```

Comment:
Not sure how much we can do here, but for scores such as this one, I think book is a much better fit that audio recording

Reply:
Okay, this comes from the RIS type `TY  - MUSIC`, but I can change it.